--- origsrc/libgweather-3.10.2/configure.ac	2014-02-02 10:38:57.000000000 -0600
+++ src/libgweather-3.10.2/configure.ac	2014-03-12 20:40:13.532784100 -0500
@@ -58,8 +58,7 @@ dnl -- check for glib; redundant at this
 AM_PATH_GLIB_2_0($GLIB_REQUIRED)
 
 dnl gettext
-AM_GNU_GETTEXT_VERSION([0.18])
-AM_GNU_GETTEXT([external])
+AM_GLIB_GNU_GETTEXT
 
 IT_PROG_INTLTOOL([0.50.0])
 IT_PO_SUBDIR([po-locations])
